spec/controllers/profile_controller_spec.rb

```diff
@@ -2,18 +2,24 @@
 
 RSpec.describe ProfileController, type: :controller do
 
+  before do
+    @user = User.create(:last_name => 'James')
```

Comment:
There was already a user created from fixtures on line 12, not sure which one it will `id` be assigned to. Not a blocker tho, since it's passing all the tests right now.